webrender/src/render_backend.rs

```diff
@@ -702,6 +718,13 @@ impl RenderBackend {
         self.last_scene_id
     }
 
+    pub fn set_render_on_generate_frame(&mut self) {
```

Comment:
As a complete outsider this seems confusing to me. If a method starts with `set` I assume I can change it to what I specify, but it always sets it to true. Wouldn't `enable_render_on_generate_frame` make more sense?

Reply:
Thanks for the advice:) Yes, it makes more sense!